packages/desktop-client/src/components/sidebar/Account.tsx

```diff
@@ -149,3 +153,88 @@ export function Account({
     </View>
   );
 }
+
+export function Account({
+  name,
+  account,
+  connected,
+  pending = false,
+  failed,
+  updated,
+  to,
+  query,
+  style,
+  outerStyle,
+  onDragChange,
+  onDrop,
+}: AccountProps) {
+  const accountNote = useNotes(`account-${account?.id}`);
+  const needsTooltip = !!account?.id;
+
+  return needsTooltip ? (
+    <Tooltip
+      content={
+        <View
+          style={{
+            padding: 10,
+          }}
+        >
+          <Text
+            style={{
+              fontWeight: 'bold',
+              borderBottom: accountNote ? `1px solid ${theme.tableBorder}` : 0,
+              marginBottom: accountNote ? '0.5rem' : 0,
+            }}
+          >
+            {name}
+          </Text>
+          {accountNote && (
+            <Notes
+              getStyle={() => ({
+                padding: 0,
+              })}
+              notes={accountNote}
+            />
+          )}
+        </View>
+      }
+      className={`${css({
+        borderRadius: '0px 5px 5px 0px ! important',
+      })}`}
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏this should solve the problem without the need of `!important`
```suggestion
      style={{ ...styles.tooltip,  borderRadius: '0px 5px 5px 0px', }}
```